src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Input/Commands/EditProjectFileCommand.cs

```diff
@@ -96,11 +115,24 @@ internal abstract class AbstractEditProjectFileCommand : AbstractSingleNodeProje
             Assumes.NotNull(textDoc);
             textDoc.FileActionOccurred += TextDocument_FileActionOccurred;
 
-            Verify.HResult(frame.Show());
+            Verify.HResult(_frame.Show());
```

Comment:
Apparently using OpenDocument after setting the default for .csproj files to an external editor still opens the file in the XML editor.